src/devices/tuya.ts

```diff
@@ -11447,6 +11447,57 @@ const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_fhvdgeuh']),
+        model: 'TS0601',
+        vendor: 'Tuya',
+        description: 'Din rail switch with power monitoring and threshold settings',
+        fromZigbee: [tuya.fz.datapoints, fz.metering],
+        toZigbee: [tuya.tz.datapoints],
+        configure: async (device, coordinatorEndpoint) => {
+            await tuya.configureMagicPacket(device, coordinatorEndpoint);
+            // Required to get the device to start reporting
+            await device.getEndpoint(1).command('manuSpecificTuya', 'dataQuery', {});
+        },
+        exposes: [
```

Comment:
A lot of exposes are missing, make sure that add one for every datapoint

Reply:
I rather remove the useless datapoints, do you any datapoint that is usefull?

Maybe a button to clear the energy, but I don't know how to make an expose for that.
Same for Fault, don't know the values
Leakage current is always 0 cause I have no device that can make that higher than 0
The reclose datapoints seems useless to me

{
    "Clear energy": false,
    "Cycle Schedule": "EwAAAAAAAAAAAA==",
    "Fault": 0,
    "Leakage current": 0,
    "Phase A": {
        "0": 9,
        "1": 50,
        "2": 0,
        "3": 0,
        "4": 0,
        "5": 0,
        "6": 0,
        "7": 0
    },
    "Switch Inching": "AAAA",
    "Switch prepayment": false,
    "Timer": 0,
    "current": 0.19,
    "current_temperature": 22,
    "energy": 4.32,
    "linkquality": 0,
    "power": 27,
    "reclose recover seconds": 30,
    "reclosing allowed times": 5,
    "reclosing enable": true,
    "relay status for power on": 2,
    "state": "ON",
    "state_l1": "ON",
    "status": 1,
    "switch": true,
    "voltage": 232.4
}